cypress/common/base.ts

```diff
@@ -568,5 +573,44 @@ export class BaseMethods {
         expect(text).to.be.eq(value)
 
     }
+
+    public checkBrowserAlertByText({
```

Comment:
this method should be already added to base. Please check for duplicate

Reply:
Fixed

Reply:
You deleted method with increased wait for alert. Please increase to 3000 in which you left